Example/Vouched/FaceViewController.swift

```diff
@@ -69,7 +70,12 @@ class FaceViewController: UIViewController {
                 self.instructionLabel.text = "Processing Image"
                 DispatchQueue.global().async {
                     do {
-                        let job = try self.session?.postFace(detectedFace: result)
+                        let job: Job?
+                        if (self.isLivenessJob) {
+                            job = try self.session?.postSelfieLiveness(detectedFace: result)
```

Comment:
Just to comment that I just realized that in the Android SDK I named that method as `postSelfieVerification` so, I think it may be strange that they have different names 😅

Reply:
Yeah, good catch - Alex and I had even discussed this, just forgot to push the change to the sdk after I made it.